src/config.h

```diff
@@ -187,6 +187,11 @@
   #undef USE_BIND
 #endif
 
+/*
+ * Include user options set from configure script.
+ */
+#include "userconf.h" /* @NO_DEP */
```

Comment:
So e.g Watcom's compiler on Windows should find this as `build/watcom/win32/userconf.h`.
I don't see where in `Makefile.all` the `I_CFLAGS = -I./build/watcom/win32` is.

Reply:
It's in regular `CFLAGS` (line 586), together with `-I. -I../inc`.

Reply:
> It's in regular CFLAGS (line 586), together with -I. -I../inc.

But from the last CI build for Watcom/Win32:
```
wcc386 @build\watcom\win32\wcc.arg accept.c -fo=build\watcom\win32\accept.obj
config.h(193): Error! E1055: Unable to open 'userconf.h'
```

So some issue with the `-I` path.

Reply:
Ahh, I had the path swapped (`watcom/build` instead of `build/watcom`).  Fixed now.

Reply:
And also in `src\test\makefile.all` since the CI says:
```
wcc386 @wcc386.arg
..\config.h(193): Error! E1055: Unable to open 'userconf.h'
make: *** [watcom_f.mak:175: bind.exe] Error 1
```
etc. 

I should fix `appveyor-build.bat` to error out in this case. This could work (for Watcom at least):
```diff
--- a/appveyor-script.bat
+++ b/appveyor-script.bat
@@ -403,9 +403,9 @@ exit /b 0
   if %BUILDER%. == cygwin.   make -f Cygwin_%BITS%.mak
   if %BUILDER%. == visualc.  make -f visualc_%BITS%.mak
   if %BUILDER%. == watcom. (
-     if %MODEL%. == large. make -f watcom_l.mak
-     if %MODEL%. == flat.  make -f watcom_f.mak
-     if %MODEL%. == win32. make -f watcom_w.mak
+     if %MODEL%. == large. (make -f watcom_l.mak & if not errorlevel == 0 goto test_failed)
+     if %MODEL%. == flat.  (make -f watcom_f.mak & if not errorlevel == 0 goto test_failed)
+     if %MODEL%. == win32. (make -f watcom_w.mak & if not errorlevel == 0 goto test_failed)
   )

   %_ECHO% "\e[1;33mRunning test 'cpu.exe' ---------------------------------------------------------------\e[0m"
@@ -417,6 +417,9 @@ exit /b 0
   %_ECHO% "\e[1;33mRunning test 'chksum.exe -s' ---------------------------------------------------------\e[0m"
   chksum.exe -s

+  exit /b 0
+
+:test_failed
+  exit /b 1
+
 :no_tests
   exit /b 0
```

Reply:
> And also in `src\test\makefile.all` since the CI says:
> 
> ```
> wcc386 @wcc386.arg
> ..\config.h(193): Error! E1055: Unable to open 'userconf.h'
> make: *** [watcom_f.mak:175: bind.exe] Error 1
> ```
> 
> etc.

Should be fixed now.